src/print.jl

```diff
@@ -444,16 +526,22 @@ struct _SolutionSummary
     barrier_iterations::Union{Missing,Int}
     simplex_iterations::Union{Missing,Int}
     node_count::Union{Missing,Int}
+    # Sensitivity
+    sensitivity::Bool
+    lp_sensitivity_summary::Union{Missing,_LPSensitivitySummary}
 end
 
 """
-    solution_summary(model::Model; verbose::Bool = false)
+    solution_summary(model::Model; verbose::Bool = false, sensitivity = false)
```

Comment:
```suggestion
    solution_summary(model::Model; verbose::Bool = false, lp_sensitivity = false)
```
since its only for LP